skbio/metadata/tests/test_io.py

```diff
@@ -1,6 +1,6 @@
 import collections
 import os.path
-import pkg_resources
+# import pkg_resources
```

Comment:
Good catch. We can remove this line (rather than just commenting it out).

Reply:
Thank you for the feedback! I have removed the line as suggested and pushed the commit.